doc/sphinx/install.rst

```diff
@@ -119,7 +124,7 @@ Finally, create a file ``dissemin/settings/__init__.py`` with this content::
 
 Depending on your environment, you might want to override ``STATIC_ROOT`` and ``MEDIA_ROOT``, in your ``__init__.py`` file. Moreover, you have to create these locations.
 
-Don't forget to edit ``ALLOWED_HOSTS`` for production.
+Don't forget to edit ``ALLOWED_HOSTS`` for production or if your django server does not run on *localhost:8000*.
```

Comment:
Default port is `8080` I think, not `8000`, cf https://github.com/dissemin/dissemin/blob/master/Vagrantfile#L28.

Reply:
Super! Thanks!